timetracker/vms/test/models/test_time_record_model.py

```diff
@@ -17,3 +20,40 @@ def test_is_approved_with_approval(time_record_approval_factory):
     record = approval.time_record
 
     assert record.is_approved
+
+
+def test_1_time_string_conversion(time_record_factory):
```

Comment:
I would name these tests with descriptions rather than numbers. Perhaps `test_string_conversion_no_end_time`, `test_string_conversion_same_date`, and `test_string_conversion_different_dates`.